python/ray/__init__.py

```diff
@@ -51,6 +51,11 @@
     os.path.abspath(os.path.dirname(__file__)), "pyarrow_files")
 sys.path.insert(0, pyarrow_path)
 
+psutil_path = os.path.join(
```

Comment:
1. psutil_path -> thirdparty_path
2. Add a comment saying "This is done to make sure we use the bundled version of psutil and setproctitle"